server/index.js

```diff
@@ -0,0 +1,129 @@
+require('dotenv').config();
+const express = require('express');
+const helmet = require('helmet');
+const cors = require('cors');
+const server = express();
+const jwt = require('jsonwebtoken');
+
+const bcrypt = require('bcryptjs');
+server.use(helmet());
+server.use(cors());
+server.use(express.json());
+const db = require('./data/dbConfig.js'); // database configuration file location
+
+// CONFIG: server settings
+const serverPort = process.env.PORT || 7100; // server port
+const serverName = `auth-i`; // Name of server to display at "/" endpoint 
+const projectPullRequest = `https://github.com/LambdaSchool/auth-ii/pull/215`;
+
+// CONFIG: endpoint routing
+// const users = require('./data/routes/usersRoutes');
+// server.use('/api/users', users);
+
+// CONFIG: Models
+const users = require('./data/models/usersModels');
+
+// ENDPOINTS
+server.get('/', (req, res) => { // sanity check root endpoint
+  res.send(`${serverName} running on port ${serverPort}<br>More information: <a href="${projectPullRequest}">GitHub Repo</a>`);
+});
+
+const jwtSecret = process.env.JWT_SECRET || 'add a secret to your .env file with this key';
+
+function generateToken(user) { // Generates token for cookie
+
+  const jwtPayload = { // payload sets the information that will be sent through the cookie (NOTE SECURE)
+    ...user,
+    hello: `${user}`,
+    role: 'admin'
+  };
+
+  jwtOptions = {
+    expiresIn: '1m'
+  };
+
+  return jwt.sign(jwtPayload, jwtSecret, jwtOptions);
+}
+
+server.post('/api/login', (req, res) => { // api login endpoint
+  const creds = req.body; // store body of post request in credentials variable
```

Comment:
Destructure the username, password, and department from the request body for database security purposes.